File: frontend/src/app/shared/models/rrhh/empleado-rol-response.ts
export interface EmpleadoRolResponse {
  idEmpleado: number;
  nombres: string;
  apellidos: string;
  numeroDocumento: string;
  celularPersonal: string;
  correoPersonal: string;
  categoriaPersonal?: 'ESTRUCTURAL' | 'OPERATIVO';
  puestoTrabajo: string;
  estadoOperativo: 'ACTIVO' | 'INACTIVO';
}

File: rrhh-service/src/main/java/pe/albrugroup/rrhh_service/entity/response/EmpleadoRolResponse.java
package pe.albrugroup.rrhh_service.entity.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import pe.albrugroup.rrhh_service.entity.enums.PuestoTrabajo;
import pe.albrugroup.rrhh_service.entity.enums.EstadoOperativo;

@Builder
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class EmpleadoRolResponse {

    private Long idEmpleado;
    private String nombres;
    private String apellidos;
    private String numeroDocumento;
    private String celularPersonal;
    private String correoPersonal;
    private PuestoTrabajo puestoTrabajo;
    private EstadoOperativo estadoOperativo;
}
